themes/default/content/docs/concepts/projects/project-file.md

```diff
@@ -154,8 +157,10 @@ Schemas are only valid for project property keys. For setting the value of a pro
 
 | Name | Required | Description |
 | - | - | - |
+| `displayName` | optional | A user-friendly name for the template. |
```

Comment:
Minor - but should we make any suggestion on format?  I think we generally are encouraging Title Case and relatively short?